docs/book/development.html

```diff
@@ -155,31 +155,31 @@ <h2 id="requirements"><a class="header" href="#requirements">Requirements</a></h
 </ul>
 <h2 id="installation"><a class="header" href="#installation">Installation</a></h2>
 <p>You need to first <em>clone</em> this repository and run:</p>
-<pre><code class="language-bash">cd zombienet
-npm install
-npm run build
+<pre><code class="language-bash">❯ cd zombienet
+❯ npm install
+❯ npm run build
 </code></pre>
 <h3 id="download-and-install-needed-artifacts-optional"><a class="header" href="#download-and-install-needed-artifacts-optional">Download and install needed artifacts (optional)</a></h3>
 <p>For an easier and faster setup of your local environment, run:</p>
-<pre><code class="language-bash">node dist/cli.js setup &lt;binaries&gt;
+<pre><code class="language-bash">❯ cd zombinet/javascript/packages && node ./cli/dist/cli.js setup &lt;binaries&gt;
```

Comment:
```suggestion
<pre><code class="language-bash">❯ cd zombinet/javascript && npm run build zombie -- setup &lt;binaries&gt;
```

Reply:
We shouldn't change the `html` files of the book, this is auto-generated right @wirednkod ?

Reply:
@pepoviola, I thought that they should be updated as they are stored at github as well